src/Barricade.cpp

```diff
@@ -5,31 +5,37 @@
 #define PI 3.14159265
 
 Barricade::Barricade(int health, int state, bool boolPlaceable, bool boolPlaced)
-                    : Object(BARRICADE_HEIGHT, BARRICADE_WIDTH),
-                      health(health), state(state),
-                      boolPlaceable(boolPlaceable), boolPlaced(boolPlaced) {
+        : Object(BARRICADE_HEIGHT, BARRICADE_WIDTH),health(health), state(state), boolPlaceable(boolPlaceable),
+        boolPlaced(boolPlaced) {
```

Comment:
boolPlaceable violates variable naming conventions.
All hungarian notation and variations thereof are forbidden.
Rename variable to isPlaced or something along those lines.
Type should never be part of the variable identifier.